### PR TITLE
Resturant page

### PR DESCRIPTION
**Pull request by jaspreet-singh-sahota**
### Feature to look for:- 
- Background video
- Google map (Used google API)
- Tab Switching functionality
- Carousel Functionality (In menu page)

Note:-  There is only one Div in the HTML page, the whole project is built from JavaScript(DOM)

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -13,7 +13,9 @@
     "no-shadow": "off",
     "no-param-reassign": "off",
     "eol-last": "off",
-    "arrow-parens": "off"
+    "arrow-parens": "off",
+    "import/extensions": "off",
+    "import/no-unresolved": "off"
```
Comment (shshamim63) [inline .eslintrc.json]: Please do not make any changes to the [config file](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).

**Review by shshamim63 — CHANGES_REQUESTED** [review]
## :recycle: Changes Required :recycle: 
___
Hi @jaspreet-singh-sahota :wave: 
Nice UI, separated modules neat and clean code :clap: But the is a major problem related to the linter. You should not make any changes to it. Please revert the change and fix the linter.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

File: .eslintrc.json (hunk 1 above)
Comment (jaspreet-singh-sahota) [reply .eslintrc.json]: I put those lines initially there, I forgot to remove it. Thanks for pointing out the issue and for your feedback on the project.

Jaspreet Singh

**Review by shshamim63 — APPROVED** [review]
## :fireworks: Approved
___
Hi @jaspreet-singh-sahota :wave: 
Congratulation :tada: you have properly completed the project. This :+1: is from me, you can go ahead.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Pull request merged**